pkg/webui/console/views/overview/index.js

```diff
@@ -247,6 +248,23 @@ export default class Overview extends React.Component {
                   component="h2"
                 />
               )}
+              <Message
+                className={style.getStarted}
+                content={m.needHelp}
+                values={{
+                  documentationLink: (
+                    <Link.Anchor secondary href={documentationBaseUrl}>
```

Comment:
This should actually be a `<Link.DocLink>`
```suggestion
                    <Link.DocLink secondary path="/" external>
```
To make this work you need to allow custom titles for DocLinks.